opt_out_http_api/run.py

```diff
@@ -1,5 +1,6 @@
 from api_methods import API
+from opt_out_http_api.store.memory import OptOutMemory
 
-app = API()
+app = API(backend_class=OptOutMemory)
```

Comment:
I would rather pass in `OptOutMemory()` -- i.e. an instance of the class -- since not every class will be constructed in the same way.